src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -4554,6 +4566,29 @@ VOID UnwindManagedExceptionPass2(PAL_SEHException& ex, CONTEXT* unwindStartConte
 
     } while (Thread::IsAddressInCurrentStack(sp) && (establisherFrame != ex.TargetFrameSp));
 
+#ifdef DEBUGGER_EXCEPTION_INTERCEPTION_SUPPORTED
+    if ((establisherFrame == ex.TargetFrameSp) && (ex.TargetIp != 0))
+    {
+#ifdef HOST_AMD64
```

Comment:
should this be `TARGET_AMD64` or `HOST_AMD64` is correct?

Reply:
HOST and TARGET are always identical when this specific code is being compiled, so it does not really matter. We tend to prefer TARGET when there is a choice.

Reply:
The mix of HOST and TARGET in this change looks arbitrary. We may want to flip everything to TARGET.

Reply:
yeah `TARGET` always reads better for me.